flink-python/pyflink/fn_execution/state_impl.py

```diff
@@ -798,6 +857,11 @@ def __init__(self,
             state_handler, map_state_read_cache_size)
         from pyflink.fn_execution.coders import FlattenRowCoder
         self._key_coder_impl = FlattenRowCoder(key_coder._field_coders).get_impl()
+        self.window_coder = window_coder
```

Comment:
I think we do not need to store the window_coder because it is not used anywhere except the init method.

Reply:
Yes. This should actually be added in the follow-up PR, I will delete it first.